src/modules/modals/CreateRow.vue

```diff
@@ -12,6 +12,10 @@
 				<ColumnFormComponent
 					:column="column"
 					:value.sync="row[column.id]" />
+				<NcNoteCard v-if="column.mandatory && !isValueValidForColumn(row[column.id], column)"
+					type="error">
+					{{ t('tables', `"${column.title}" should not be empty`) }}
```

Comment:
```suggestion
					{{ t('tables', '"{columnTitle}" should not be empty', { columnTitle: column.title }) }}
```

Placeholders need to passed in separately for translations so they can be handled by the translators.

Reply:
I don't know what "passed in separately" means here

Reply:
The suggested code change shows how to do it.

The idea is that the `t()` function will do the parameter replacement instead of directly doing it in the string passed over. That way translators will be able to see and translate the text with the placeholder value still preserved and it is only inserted at runtime.

Reply:
https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#javascript

Reply:
I see, thanks